allennlp/models/encoder_decoders/wikitables_semantic_parser.py

```diff
@@ -450,25 +448,28 @@ def _get_linking_probabilities(worlds: List[WikiTablesWorld],
                                                  dim=1,
                                                  index=Variable(linking_scores.data.new(row_type_index)).long())
 
-            # (num_question_tokens, num_entities_per_type)
+            # (num_question_tokens, num_entities_per_type + 1)
```

Comment:
Is the `+1` for the null entities? A comment would be nice. Also update above the lines with index_select`.

Reply:
Done.